{
  "comments": [
    {
      "key": {
        "uuid": "365aa3b4_c3acb97e",
        "filename": "mongo/errors.go",
        "patchSetId": 2
      },
      "lineNbr": 276,
      "author": {
        "id": 1001503
      },
      "writtenOn": "2019-06-26T14:47:13Z",
      "side": 1,
      "message": "Instead of having these as standalone functions, can we add them as methods on the CommandError type?",
      "range": {
        "startLine": 276,
        "startChar": 5,
        "endLine": 276,
        "endChar": 28
      },
      "revId": "9dc1542a1b45bacea335fa4133c3edbaf34735fb",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}